README.md

```diff
@@ -19,7 +19,12 @@ Links between docs can be achieved by using a hash symbol (#), plus the markdown
 ### Alert Boxes
 Alert boxes are achieved by using a block quote that has one of 'warn', 'danger' or 'info' on the first line.
 
-## Join the Unofficial Discord API Server
+## Need some help?
+Here are some Discord servers that can help you out with everything Discord API:
+
 The Unofficial Discord API server is a common hangout for library and bot developers alike. It's a great starting point for those looking to dive in and learn bot-creation with the Discord API.
```

Comment:
Should consider removing this as it was mentioned by discord staff in a previous PR. https://github.com/discord/discord-api-docs/pull/1792#issuecomment-655115052

Reply:
The unofficial server is still by far the best place to get help with creating bots and libraries.

Reply:
> The unofficial server is still by far the best place to get help with creating bots and libraries.
This is true, however the official server is much better for things like GameSDK, and other Discord Developer related things, mentioned in the docs.

Reply:
It should definitely stay in my opinion. The official server cannot help with specific library questions so they often redirect them to Discord API anyway.

Reply:
No need to remove DAPI--definitely still the place to go for library help